zengo/models.py

```diff
@@ -84,7 +84,9 @@ class Ticket(models.Model):
         Constant(normal="normal"),
         Constant(low="low"),
     )
-    priority = ConstantChoiceCharField(constants=priorities, max_length=8)
+    priority = ConstantChoiceCharField(
+        constants=priorities, max_length=8, null=True, blank=True
```

Comment:
If you are allowing nulls, I think `blank=True` is not necessary, is it?

Reply:
It is only really needed to keep the admin happy when editing/adding a Ticket

Reply:
Otherwise you can't do so without adding a non-null value